src/probnum/quad/solvers/policies/_max_aquisition_policy.py

```diff
@@ -69,3 +70,87 @@ def __call__(
         values = self.acquisition_func(random_nodes, bq_state)[0]
         idx_max = int(np.argmax(values))
         return random_nodes[idx_max, :][None, :]
+
+
+class MaxAcquisitionPolicy(Policy):
+    """Policy that maximizes an acquisition function with an optimizer.
+
+    Parameters
+    ----------
+    batch_size
+        Size of batch of nodes when calling the policy once (must be equal to 1).
+    acquisition_func
+        The acquisition function.
+    n_restarts
+        The number of times the optimizer is being restarted.
+
+    Raises
+    ------
+    ValueError
+        If ``batch_size`` is not 1, or if ``n_restarts`` is too small.
+
+    Notes
+    -----
+    The policy uses SciPy's 'Nelder-Mead' optimizer when gradients are unavailable.
+    This is the current standard setting since ``probnum`` does not provide gradients
+    yet.
+
+    """
+
+    def __init__(
+        self,
+        batch_size: IntLike,
+        acquisition_func: AcquisitionFunction,
+        n_restarts: IntLike,
+    ) -> None:
+
+        if batch_size != 1:
+            raise ValueError(
+                f"MaxAcquisitionPolicy can only be used with batch size 1 "
+                f"({batch_size})."
+            )
+        if n_restarts < 1:
+            raise ValueError(
+                f"The number of restarts ({n_restarts}) must be equal to"
+                f"or larger than 1."
+            )
+
+        super().__init__(batch_size=batch_size)
+        self.acquisition_func = acquisition_func
+        self.n_restarts = int(n_restarts)
+
+    @property
+    def requires_rng(self) -> bool:
+        return True
+
+    def __call__(
+        self, bq_state: BQState, rng: Optional[np.random.Generator]
+    ) -> np.ndarray:
+
+        measure = bq_state.measure
+        domain = bq_state.measure.domain
+        bounds = list(zip(domain[0], domain[1]))
+
+        if self.acquisition_func.has_gradients:
+            # Todo (#581): Gradients should not be available yet.
+            raise NotImplementedError
+
+        f = lambda x: -self.acquisition_func(x[None, :], bq_state)[0][0]
+        initial_points = measure.sample(n_sample=self.n_restarts, rng=rng)
+        x_min, f_min, success = None, np.inf, False
```

Comment:
Since `f` is non-positive, should we have `f_min = -np.Inf` instead (with a corresponding change on line 145)?

Reply:
Not sure I understand. $f(x)$ needs to be maximized but standard optimizers always minimize. Hence, in line138 the sign of $f(x)$ is flipped. The initial value of `f_min` is set to plus infinity and is replaced whenever a smaller value is found after a single run of the optimizer. Accordingly, the current best guess `x_min` is updated with the value corresponding to `f_min`. Maybe I am not seeing something? 

The code should be equivalent to running the optimizer on $-f(x)$ `initial_points` times and then returning the minimizer with the smallest function value. 

There are no assumptions on the values $f(x)$ inside this method.

Reply:
Sorry - it seems I got confused.